sweep/sweeper.go

```diff
@@ -52,11 +52,22 @@ type Params struct {
 
 	// Force indicates whether the input should be swept regardless of
 	// whether it is economical to do so.
+	//
+	// TODO(yy): Remove this param once deadline based sweeping is in place.
```

Comment:
Yeah I think I'll rename this to `Immediate` instead of `Force`, on the other hand, with the new fee bumper, we can build RPCs to manually increase its priority.